bin/install_swiftpm_dependencies

```diff
@@ -111,15 +119,15 @@ elif [[ "${USE_SPM}" == "true" ]]; then
   swift package resolve
 fi
 
-
-# `checkouts` can be removed because the system can quickly generate them 
+# `checkouts` can be removed because the system can quickly generate them
 # instead of needing to download them in the cache each time.
-# 
-# `artifacts` should be removed because it causes issues when downloading 
-# certain packages to have the artifacts already present after extracting 
+#
+# `artifacts` should be removed because it causes issues when downloading
+# certain packages to have the artifacts already present after extracting
 # cache
 echo "~~~ Cleaning up cache files before saving cache"
 rm -rf "${SPM_CACHE_LOCATION}/checkouts" "${SPM_CACHE_LOCATION}/artifacts"
+echo "Done. Removed checkouts and artifacts subfolders from $SPM_CACHE_LOCATION"
```

Comment:
Noticed this nitpick while checking the logs in the integration tests...

Before (without `echo`)

<img width="358" alt="image" src="https://github.com/user-attachments/assets/e1715fbd-780c-4b8e-a18e-a724542ed879">

After

<img width="835" alt="image" src="https://github.com/user-attachments/assets/299d438e-478d-4c46-8f69-4d8c1444d504">